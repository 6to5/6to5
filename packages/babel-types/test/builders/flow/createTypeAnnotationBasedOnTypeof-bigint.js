import { createTypeAnnotationBasedOnTypeof } from "../../..";

describe("builders", function () {
  describe("flow", function () {
    (parseFloat(process.versions.node) >= 12 ? describe : describe.skip)(
      "createTypeAnnotationBasedOnTypeof on bigint",
      function () {
        const values = {
          bigint: typeof 0n,
        };

        for (const name in values) {
          const value = values[name];
          it(name, function () {
            const result = createTypeAnnotationBasedOnTypeof(value);
            expect(result).toMatchSnapshot();
          });
        }
      },
    );
  });
});
