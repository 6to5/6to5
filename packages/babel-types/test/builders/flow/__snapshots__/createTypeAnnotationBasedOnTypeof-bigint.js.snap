// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`builders flow createTypeAnnotationBasedOnTypeof on bigint bigint 1`] = `
Object {
  "type": "AnyTypeAnnotation",
}
`;
